flexget/api.py

```diff
@@ -22,41 +25,111 @@ def attach_schema(response):
     return response
 
 
-@api_schema.route('/version')
-def version_schema():
-    return jsonify({
-        'type': 'object',
-        'properties': {
-            'flexget_version': {'type': 'string', 'description': 'FlexGet version string'},
-            'api_version': {'type': 'integer', 'description': 'Version of the json api'}
-        }
-    })
-
-
-@api.route('/version')
+@api.route('/version/')
 def version():
     return jsonify(flexget_version=flexget.__version__, api_version=API_VERSION)
 
 
+# Execution API
+
 exec_parser = get_parser('execute')
 
+def _get_task(task_id):
```

Comment:
I think if we are going to be looking anything up about task executions after the fact, we need to store the info here. Such that when someone calls the /execution/ endpoint we generate and store the unique_id -> task info mapping here. I'm not sure how long we need to hold on to that though...